src/pipeline_factory.cpp

```diff
@@ -57,4 +58,16 @@ Status PipelineFactory::create(std::unique_ptr<Pipeline>& pipeline,
     lock.unlock();
     return definition.create(pipeline, request, response, manager);
 }
+void PipelineFactory::revalidatePipelines(ModelManager& manager) {
+    for (auto& [name, d] : definitions) {
+        if (d->getStatus().requireRevalidation()) {
+            auto validationResult = d->validate(manager);
+            if (!validationResult.ok()) {
+                SPDLOG_LOGGER_ERROR(modelmanager_logger, "Revalidation pipeline definition: {} failed: {}", name, validationResult.string());
```

Comment:
You put validation status string into log, are you sure we have all pipeline validation statuses mapped to some string?

Reply:
We print those statuses in normal pipeline validation (line 36)

Reply:
It's extracting the status string the same way as when creating pipeline definition. 

Conclusion here is that we omitted by accident some of pipeline definition statuses, and even when they are reported in logs it's still useful to have them mapped in status codes string map.

Reply:
Added missing status to string mappings.